.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "atmos_cubed_sphere"]
   path = atmos_cubed_sphere
-  url = https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere
-  branch = dev/emc
+  url = https://github.com/SamuelTrahanNOAA/GFDL_atmos_cubed_sphere
+  branch = lightning
```

Comment:
This still need to be updated/reverted if the atmos_cubed PR was merged.

Reply:
I have reverted it.